src/Morphir/SDK/Basics.elm

```diff
@@ -0,0 +1,4 @@
+module Morphir.SDK.Basics exposing (Int, Int32)
+
+type alias Int32 = Basics.Int
+type alias Int = Int32
```

Comment:
Is this sufficient for the `Basics` definition or do we actually want to repeat the full `elm/core` definition as well?

Reply:
This is sufficient. Can you remove the `Int` alias? We only need one of those.